proprietary_secure_channel/README.md

```diff
@@ -0,0 +1,245 @@
+# The Remote Attestation Sample
+
+This sample demonstrates how to do remote attestation between two enclaves.
+
+It has the following properties:
+
+- Written in C++
+- Demonstrates an implementation of remote attestation
+- Use of mbedTLS within the enclave
+- Use Asymmetric / Public-Key Encryption to establish secure communications between two attesting enclaves
+- Enclave APIs used:
+  - oe_get_report
+  - oe_verify_report,
+  - oe_is_within_enclave
+
+**Note: Currently this sample only works on SGX-FLC systems.** The underlying SGX library support for end-to-end remote attestation is available only on SGX-FLC system. There is no plan to back port those libraries to either SGX1 system or software emulator.
+
+## Attestation primer
```

Comment:
To avoid out-of-sync between duplicated copies of information, this whole Attestation Primer could be replaced by a link to the OS SDK samples.

Reply:
Sounds good. Will have some basic info and then point to the OE SDK samples.

Reply:
I am going to leave the Attestation primer here as the information is correct and may not get stale. It is short as well.

Reply:
Fixed this and also removed some more redundant parts pertaining to remote attestation details.